runtime/manual/references/contributing/building_from_source.md

```diff
@@ -48,10 +48,12 @@ cargo -V
 > Many components of Deno require a native compiler to build optimized native
 > functions.
 
-**Linux**:
-
-```sh
-apt install --install-recommends -y clang-16 lld-16 cmake libglib2.0-dev
+**Linus/WSL**
```

Comment:
`Linus` -> `Linux` here

Reply:
@kwhinnery its better if we use `Debian` instead of Linux? since apt is supported only in Debian based linux I guess.

Reply:
That would probably be ideal, yes

Reply:
@kwhinnery made the changes like below
Linux -> Linux(Debian)